### PR TITLE
Add TransTECF411AIO target

### PR DESCRIPTION
**Pull request by flosean**
Add TransTEC F411 AIO new board target.
This is an AIO board that design for specific Frame, and default setting of ESC support RPM filter.

### CODE REVIEW

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+aux 1 1 2 1300 1700 0 0
+aux 2 13 1 1700 2100 0 0
+aux 3 35 0 1700 2100 0 0
+
+# master
+set dyn_notch_width_percent = 0
+set dyn_notch_q = 250
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = NONE
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:110]: This should not be set explicitly - the default will work just fine if no hardware is found.

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set baro_hardware = NONE
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:113]: This should not be set explicitly - the default will work just fine if no hardware is found.

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+# aux
+aux 0 0 2 1300 2100 0 0
+aux 1 1 2 1300 1700 0 0
+aux 2 13 1 1700 2100 0 0
+aux 3 35 0 1700 2100 0 0
+
+# master
+set dyn_notch_width_percent = 0
+set dyn_notch_q = 250
+set mag_bustype = I2C
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:108]: Why are you configuring a bus for MAG if you do not have a MAG in your product?

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+resource MOTOR 5 B03
+resource MOTOR 6 B10
+resource PPM 1 A03
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource INVERTER 1 A14
+resource I2C_SCL 1 B08
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:21]: Seems to be a bit pointless to configure an I2C bus on an RTF model that does not use any I2C devices.

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = NONE
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set baro_hardware = NONE
+set min_check = 1030
+set max_check = 1995
+set serialrx_provider = SBUS
+set blackbox_device = NONE
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:117]: Please do not explicitly specify defaults in here.

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+set dshot_bitbang = OFF
+set motor_pwm_protocol = DSHOT300
+set motor_poles = 12
+set vbat_max_cell_voltage = 450
+set current_meter = ADC
+set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set yaw_motors_reversed = ON
+set small_angle = 90
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:130]: This is a safety setting - explicitly disabling this in a target configuration is not permissible.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/TTRH-TRANSTECF411AIO.config (hunk 1 above)
Comment (flosean) [reply configs/default/TTRH-TRANSTECF411AIO.config]: Ok, I will fix this problem

File: configs/default/TTRH-TRANSTECF411AIO.config (hunk 3 above)
Comment (flosean) [reply configs/default/TTRH-TRANSTECF411AIO.config]: Sorry, I didn't notice it, will remove it.

File: configs/default/TTRH-TRANSTECF411AIO.config [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+led 1 1,0::L:1
+
+# aux
+aux 0 0 2 1300 2100 0 0
+aux 1 1 2 1300 1700 0 0
+aux 2 13 1 1700 2100 0 0
+aux 3 35 0 1700 2100 0 0
+
+# master
+set baro_bustype = I2C
```
Comment (mikeller) [inline configs/default/TTRH-TRANSTECF411AIO.config:104]: Naturally, all of the `baro` settings should be removed as well if baro is not to be used.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/TTRH-TRANSTECF411AIO.config (hunk 7 above)
Comment (flosean) [reply configs/default/TTRH-TRANSTECF411AIO.config]: Ok, I will remove it.

**Review by mikeller — APPROVED** [review]

**Pull request merged**